tools/setup/emoji/build_emoji

```diff
@@ -110,14 +124,15 @@ def generate_sprite_css_files(cache_path: str,
             # those google emoji (in case anyone used them).
             emoji_positions += EMOJI_POS_INFO_TEMPLATE % {
                 'codepoint': get_emoji_code(emoji),
-                'pos_x': (emoji["sheet_x"] * 100) / max_dim,
-                'pos_y': (emoji["sheet_y"] * 100) / max_dim,
+                'pos_x': percent(emoji["sheet_x"] / max_dim),
+                'pos_y': percent(emoji["sheet_y"] / max_dim),
```

Comment:
Accounting for gutters, I think this actually needs to be `percent((emoji["sheet_x"] * 66 + 1) / (max_dim * 66 + 2))`.

Reply:
I re-pushed so that I no longer add that 66/64 factor for background-size, which would have been a functionality change, and which would have required the adjustment you mention.  In doing that I also killed off `max_dim` and added more comments explaining various calculations.